gr-blocks/python/blocks/qa_msg_pair_to_var.py

```diff
@@ -36,46 +34,76 @@ def test_f(value: int = 0):
 
         dut = blocks.msg_pair_to_var(test_f)
         test_msg = pmt.cons(pmt.intern("foo"), pmt.from_uint64(canary))
-        src = blocks.message_strobe(test_msg, 10)
+        src = blocks.message_strobe(test_msg, 50)
         self.tb.msg_connect(*(src, pmt.intern("strobe")),
                             *(dut, blocks.msg_pair_to_var.IN_PORT))
+
+        start_time = time.time_ns()
```

Comment:
Probably better to use a monotonic clock. And lucky for us, GNU Radio now requires Python 3.7, so we can use `monotonic_ns`.
```suggestion
        start_time = time.monotonic_ns()
```